package.json

```diff
@@ -34,7 +34,7 @@
     "postcss-nesting": "^13.0.0",
     "postcss-opacity-percentage": "^2.0.0",
     "three": "~0.167.1",
-    "vite": "^5.3.5",
+    "vite": "^5.4.0",
```

Comment:
Can you revert this change? 

It is better to focus PR on a single task (also lock file changes could bring attacks).